.github/actions/install-crane/action.yaml

```diff
@@ -0,0 +1,37 @@
+name: "Install crane"
+description: "Installs crane for use in testing."
+inputs:
+  crane-version:
+    description: "Version of crane to install"
+    required: false
+    default: latest
+  install-dir:
+    description: "Directory to install crane"
+    required: false
+    default: $HOME/.crane
+
+runs:
+  using: "composite"
+  steps:
+    - shell: bash
+      run: |
+        mkdir -p ${{ inputs.install-dir }}
+
+        VERSION=${{ inputs.crane-version }}
+        if [[ "${VERSION}" == "latest" ]]; then
+          VERSION=$(curl -s "https://api.github.com/repos/google/go-containerregistry/releases/latest" | jq -r '.tag_name')
```

Comment:
If this is an API that might be rate limited, it could be better to use the [gh CLI](https://cli.github.com/manual/gh_release_list) that's installed ([example](https://github.com/bottlerocket-os/bottlerocket-core-kit/blob/7c4561abc5582872a112f6b35ee64f4d0aca8320/.github/workflows/cache.yml#L43).)